lib/FunctionModuleTemplatePlugin.js

```diff
@@ -10,8 +10,8 @@ class FunctionModuleTemplatePlugin {
 	apply(moduleTemplate) {
 		moduleTemplate.plugin("render", function(moduleSource, module) {
 			const source = new ConcatSource();
-			const defaultArguments = [module.moduleArgument || "module", module.exportsArgument || "exports"];
-			if((module.arguments && module.arguments.length !== 0) || module.hasDependencies(d => d.requireWebpackRequire !== false)) {
```

Comment:
It's reasonable to have that thought.

However, I believe that there is no one using these properties (in reasonable way). On one hand, there is **no value assignments** of these properties in the repo. On the other, after searching through github with the keywords, there is no usage of them explicitly.

There we can conclude that, no public plugins are using them if they using the latest version of webpack. (99%)